### PR TITLE
Add class

### PR DESCRIPTION
**Pull request by rugoncalves**
This PR is for improving the code used when the app is within the preview in devices.

### CODE REVIEW

File: src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+
+		/**
+		 * Adds a css class to the body, to make it easily style accordingly.
+		 *
+		 * @private
+		 * @static
+		 * @memberof OnPostMessage
+		 */
+		private static _addInPreviewCssClass(): void {
+			OnPostMessage._isInPreviewInDevices = true;
```
Comment (BenOsodrac) [inline src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts:27]: Shouldn't this be at the end of the _messageFromPreview() method?

Comment (BenOsodrac) [reply src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts:27]: Feels weird to set it as true, on a method that only adds a class

Comment (rugoncalves) [reply src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts:27]: Can be, there's no specific advantage in either of the ways.

File: src/scripts/OSFramework/Helper/Device.ts [hunk 2]
```diff
@@ -127,16 +127,17 @@ namespace OSFramework.Helper {
 		 * @memberof DeviceInfo
 		 */
 		private static _getUserAgent(userAgent = ''): string {
-			const cleanedUserAgent = userAgent.replace(' ', '');
+			let localUserAgent = userAgent;
 
-			if (cleanedUserAgent === '') {
+			if (userAgent.replace(' ', '') === '') {
```
Comment (gnbm) [inline src/scripts/OSFramework/Helper/Device.ts:132]: Could also be `userAgent.replace(/\s/g, "")`?!

**Review by BenOsodrac — APPROVED** [review]

**Pull request closed without merging**